### DISCUSSION TITLE
Doesn't fully uninstall(?)

### DISCUSSION THREAD

**Discussion by sirhazmat** [SpotX-Official/SpotX#412] [General]
Tried both the uninstall options but neither seem to fully uninstall the adblocker to allow for a fresh install of standard spotify (with ads) 

after trying both and restarting PC between + reinstalling standard spotify it still seems to be blocking ads, banner on the bottom is blank or has random numbers/letters and no 30 second ads play 

can't seem to make sense of it, have tried uninstalling spotify via add/remove programs too and deleting the folders in appdata local + roaming but it still has something going on somewhere when spotify is reinstalled.. ads still blocked(?)

**Comment by amd64fox** [comment]
Where did you get the idea that you previously had a filled banner, etc.?
Spotx blocks ads locally, so if you've been running patches to remove it, I can be 200% sure that it simply doesn't exist on your PC physically, and even more so, it cannot block ads in your Spotify client. (Unless you are using a different ad-blocking method.)

In the Spotify system, there is a concept of non-relevant geolocation. For example, in my case, I live in Russia, and Spotify does not officially support this region, so it doesn't show ads to users with Russian IP addresses. You can check your situation by simply changing your region using a VPN, for example, to Germany. After doing that, you will see the ads.
